Example/meta.yaml

```diff
@@ -1,7 +1,6 @@
-{ "display_name":"Example Python Plugin",
-  "description": "A hello world of collectd python plugins.",
-  "code": "https://github.com/signalfx/collectd-example",
-  "featured": true,
-  "logo_small": "https://myavantiservices.files.wordpress.com/2015/02/helloworld.gif",
-  "logo_large": "https://myavantiservices.files.wordpress.com/2015/02/helloworld%402x.gif"
-}
+code: https://github.com/signalfx/collectd-example
```

Comment:
For some reason I thought the UI consumed these as JSON somehow which is why it used the JSON subset of YAML. Is that no longer the case or am I just confused?